BUILD

```diff
@@ -21,6 +21,8 @@ licenses(["notice"])  # Apache 2.0
 
 exports_files(["LICENSE"])
 
+exports_files(["WORKSPACE"])
```

Comment:
is this necessary for the tests to work? If so, add it to docs for the rule.

Reply:
Done